stan/math/rev/functor/coupled_ode_system.hpp

```diff
@@ -296,7 +297,7 @@ struct coupled_ode_system<F, var, double> {
         dz_dt[offset + i] = temp_deriv;
       }
 
-      nested.set_zero_all_adjoints();
+      set_zero_all_adjoints_nested();
```

Comment:
Is `set_zero_all_adjoints_nested();` preferable for some reason?

Reply:
Eh I think at one point I was trying to chomp off all the unneeded bits I could possibly get rid of and did this because of indirection or something but I'll switch it back to nested

Reply:
I was just curious is there was a reason to do the one over the other. If not, no need to prefer one over the other.